.document

```diff
@@ -1,3 +1,5 @@
 README.md
 ext/date/*.[ch]
 lib/date.rb
+doc/date/calendars.rdoc
```

Comment:
This will allow us to create new files in `doc/date` without needing to modify this again:

```suggestion
doc/date/*
```